script/binary

```diff
@@ -20,15 +20,20 @@ GIT_COMMIT=$(git describe --tags --dirty)
 BUILD_FLAGS=(-ldflags="-w -X github.com/kubeless/kubeless/pkg/version.Version=${GIT_COMMIT}")
 
 # Get rid of existing binary
-rm -f ${GOPATH%%:*}/bin/kubeless*
+echo "Removing Old Kubeless binary"
+rm -f ${GOROOT%%:*}/bin/kubeless
+rm -f ${GOROOT%%:*}/bin/kafka-trigger-controller
+rm -f ${GOROOT%%:*}/bin/kubeless-controller-manager
+rm -f ${GOROOT%%:*}/bin/nats-trigger-controller
```

Comment:
Note that `kafka` `kubeless` and `nats` controllers are not built with this script but with `script/binary-controller` instead.

Reply:
@andresmgot ok,I will rewrite my pr

Reply:
```
# Build binary
go install \
    "${BUILD_FLAGS[@]}" \
    github.com/kubeless/kubeless/cmd/...
```

i notice this command will build kubelss,kafka-trigger-controller,kubeless-controller-manager,nats-trigger-controller. before build,i remove all old binary...

Reply:
I wiil rewrite this script make sure all binary put under GOPATH